functions.php

```diff
@@ -69,3 +69,47 @@ function dash_restrict_ip() {
         }
     }
 }
+
+function dash_is_env( $url ) {
+
+    if ( substr( $url, 0, 4 ) === "dev-" ) {
+
+        return 'dev';
+    }
+    if ( substr( $url, 0, 5 ) === "test-" ) {
+
+        return 'test';
+    }
+    if ( substr( $url, 0, 10 ) === "editorial-" ) {
+
+        return 'editorial';
+    }
+    if ( substr( $url, 0, 9 ) === "localhost" ) {
```

Comment:
Could we possible target the port for dev environment rather than just localhost? E.g. tna-base:8888, localhost:8000 etc...

Reply:
Turned this into an issue